Add Sky Follower Bridge to projects (#95)
bluesky-social/atproto-website: content/community/projects.md

- [Twitter-to-Bsky](https://github.com/ianklatzco/twitter-to-bsky), to import your Twitter archive to Bluesky by [@klatz.co](https://bsky.app/profile/klatz.co)
- [bluesky-post](https://github.com/myConsciousness/atproto.dart/tree/main/packages/bluesky_post), a tool to post from GitHub Actions to Bluesky, by [@shinyakato.dev](https://bsky.app/profile/shinyakato.dev)
- [granary](https://granary.io/), converts `app.bsky` objects to/from ActivityStreams, RSS, Atom, HTML, and more, by [@snarfed.org](https://bsky.app/profile/snarfed.org)

> - [Sky Follower Brdige](https://github.com/kawamataryo/sky-follower-bridge), Instantly find and follow the same users from your Twitter follower on Bluesky, by [@kawamataryo.bsky.social](https://bsky.app/profile/kawamataryo.bsky.social)

Reviewer: ```suggestion
- [Sky Follower Bridge](https://github.com/kawamataryo/sky-follower-bridge), instantly find and follow Twitter followers on Bluesky, by [@kawamataryo.bsky.social](https://bsky.app/profile/kawamataryo.bsky.social)
```
